app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailsActivity.java

```diff
@@ -50,20 +54,24 @@ protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_event_details);
 
-        ButterKnife.bind(this);
+        Event initialEvent = getIntent().getParcelableExtra(EVENT_KEY);
+
+        if(initialEvent == null) {
```

Comment:
Same thing with id. First, the code was assuming that it has been given the event id and not even checking its validity and the default id was 0. There is hopefully no event with id 0, or else it would have shown its info if id wasn't passed and a whole array of authentication errors which would have made the user confused.

And even if there's no event with ID 0, it shouldn't have executed normally